python/social_bot/envs/simple_navigation.py

```diff
@@ -155,7 +155,8 @@ def reset(self):
         return obs
 
     def get_camera_observation(self):
-        image = self._agent.get_camera_observation("camera")
+        image = self._agent.get_camera_observation(
+            "default::pioneer2dx_noplugin::camera::camera")
```

Comment:
For some unknown reason: I got the following errors on my machine (Ubuntu 18.04, Gazebo 10):
[Msg] Waiting for master.
[Msg] Connected to gazebo master @ http://127.0.0.1:11345
[Msg] Publicized address: 10.40.0.7
[Err] [LogRecord.cc:158] LogRecord has not been initialized.
[Err] [SensorManager.cc:296] Unable to create sensor of type[pioneer2dx_noplugin::camera]
[Wrn] [msgs.cc:1832] Conversion of sensor type[pioneer2dx_noplugin::camera] not supported.
 joint name: pioneer2dx::pioneer2dx_noplugin::camera_joint
 joint name: pioneer2dx::pioneer2dx_noplugin::left_wheel_hinge
 joint name: pioneer2dx::pioneer2dx_noplugin::right_wheel_hinge
INFO:social_bot.envs.simple_navigation:joint names: ['pioneer2dx::pioneer2dx_noplugin::camera_joint', 'pioneer2dx::pioneer2dx_noplugin::left_wheel_hinge', 'pioneer2dx::pioneer2dx_noplugin::right_wheel_hinge']
 joint name: pioneer2dx::pioneer2dx_noplugin::camera_joint
 joint name: pioneer2dx::pioneer2dx_noplugin::left_wheel_hinge
 joint name: pioneer2dx::pioneer2dx_noplugin::right_wheel_hinge
unable to find sensor: default::pioneer2dx_noplugin::camera::camera
Segmentation fault (core dumped)

Reply:
I also have this same problem on my Ubuntu machine.

Reply:
Fixed.